Makefile

```diff
@@ -21,17 +21,17 @@ ci:
 	@if [ "$(TEST_SUITE)" = "linter" ]; then make ci-linter; else make ci-tests; fi
 
 ci-tests:
-	go get -u github.com/go-lintpack/lintpack/...
+	go get github.com/go-lintpack/lintpack/...
 	go test -v -race -count=1 -coverprofile=coverage.out ./...
 
 ci-linter:
 	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b ${GOPATH}/bin v1.12.3
 	golangci-lint run -v
-	go get -u github.com/Quasilyte/go-consistent
+	go get github.com/Quasilyte/go-consistent
```

Comment:
You could add `github.com/Quasilyte/go-consistent` as a [tool dependency](https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module); then simply `go install github.com/Quasilyte/go-consistent`